util/display/messages.go

```diff
@@ -257,6 +257,16 @@ https://guides.nanobox.io/
 `))
 }
 
+func InvalidBoxfile() {
+	os.Stderr.WriteString(fmt.Sprintf(`
+--------------------------------------------------------------------------------
+INVALID BOXFILE.YML
+Nanobox requires valid yaml in your boxfile.yml config file. Paste the contents
+of your boxfile into yamllint.com to validate.
```

Comment:
I generally try to add a "please" whenever asking users to do something. In this case, "Please paste the..."

Also, I would probably use the full domain: www.yamllint.com

Looks good!